src/seed/seed-database.ts

```diff
@@ -0,0 +1,54 @@
+import dotenv from 'dotenv';
+import { faker } from '@faker-js/faker';
+import { UserInput } from '../model/user.model.js';
+import { User } from '../data/db/index.js';
+
+dotenv.config({ path: 'test.env' });
+
+interface SeedDatabaseParams {
+  users?: {
+    name?: string;
+    email?: string;
+    password?: string;
+    birthDate?: string;
+  }[];
+}
+
+export async function seedDatabase(params?: SeedDatabaseParams) {
+  const start = performance.now();
+
+  const { AppDataSource } = await import('../data-source.js');
+  await AppDataSource.initialize();
+  const userRepository = AppDataSource.getRepository(User);
+
+  const numberOfUsers = !!params?.users?.length ? params.users.length : DEFAULT_NUMBER_OF_USERS;
+
+  for (let i = 0; i < numberOfUsers; i++) {
+    const randomUser = createRandomUser();
+    const user = {
+      name: params?.users?.[i].name ?? randomUser.name,
+      email: params?.users?.[i].email ?? randomUser.email,
+      password: params?.users?.[i].password ?? randomUser.password,
+      birthDate: params?.users?.[i].birthDate ?? randomUser.birthDate,
+    };
```

Comment:
o que acha?
```suggestion
    const randomUser = createRandomUser();
    const user = Object.assign(randomUser, params?.users?.[i])
```
Pode usar destruct tb
```suggestion
    const randomUser = createRandomUser();
    const user = { ...randomUser, ...(params?.users?.[i] ?? {}) };
```